src/helpers/tableNames.ts

```diff
@@ -0,0 +1,3 @@
+export const TableNames = {
+  USER: process.env.USER_TABLE_NAME || '',
```

Comment:
if you trying to avoid mutations, use Object.freeze

Reply:
Somehow?
```typescript
export const TableNames = Object.freeze({
  USER: process.env.USER_TABLE_NAME ?? '',
});
```